<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Setting</class>
 <widget class="QWidget" name="GUI_Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QGroupBox" name="manuals">
       <property name="title">
        <string>Manual loading</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="load_dll_from_conf_btn">
            <property name="text">
             <string>Load DLL from 
config file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="load_dll_man_btn">
            <property name="text">
             <string>Load dll 
manualy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="load_gpib_from_conf_btn">
            <property name="text">
             <string>Load GPIB variables 
from config file</string>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="load_gpib_man_btn">
            <property name="text">
             <string>Load GPIB 
manualy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="load_excel_from_conf_btn">
            <property name="text">
             <string>Load Excel file 
from config file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="load_excel_man_btn">
            <property name="text">
             <string>Load Excel 
manualy</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="Excels">
       <property name="title">
        <string>Excel settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="excel_all_tests">
              <property name="text">
               <string>All</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="excel_between_tests">
              <property name="text">
               <string>Range</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="excel_one_test">
              <property name="text">
               <string>One test</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QCheckBox" name="excel_repeat_cbx">
                <property name="text">
                 <string>Repeat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="excel_repeat_spin">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>10000000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>from</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="excel_test_from_line">
                <property name="inputMask">
                 <string>90000000</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>to</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="excel_test_to_line">
                <property name="inputMask">
                 <string>90000000</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>number:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="excel_test_num_line">
                <property name="inputMask">
                 <string>90000000</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Exclude tests (ranges throught '-' symbol, space to split)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="excel_exclude_test_line">
            <property name="placeholderText">
             <string>Write exception test numbers</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Paused tests (space to split)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="excel_pause_items">
            <property name="inputMask">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Write test numbers</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="excel_save_settings_btn">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="excel_clear_seq_btn">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="excel_start_btn">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="Starts">
       <property name="title">
        <string>Start application settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="load_dll_from_start_cbx">
          <property name="text">
           <string>Load .dll from config file when starts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="load_gpib_from_start_cbx">
          <property name="text">
           <string>Load GPIB variables from config file when starts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="load_excel_from_start_cbx">
          <property name="text">
           <string>Open Excel file from config file when starts</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Debugger settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="results_to_globvar">
              <property name="text">
               <string>Save result of test as a 
global variable. Variable 
would be called 
test_/*test number*/</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QPushButton" name="select_file_test_btn">
              <property name="text">
               <string>Select test file 
for run manualy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="info_msg_cbx">
              <property name="text">
               <string>Show info messages</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="warning_msg_cbx">
              <property name="text">
               <string>Show warning messages</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="delete_test_btn">
            <property name="text">
             <string>Delete test from a local
memory (save field empty
to delete all saved tests)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="test_for_delete_line">
            <property name="placeholderText">
             <string>Script name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
